### ISSUE TITLE
Reason for significant accuracy drop on quantize openvino ir

### ISSUE THREAD

**Issue by yogeshReflexionai** [openvinotoolkit/openvino_notebooks#837]
Hi,
I have observed 40-45% accuracy drop while performing post quantization on Residual Masking network. 
For pytorch model we have accuracy on 80%, for ir convertion we get exactly 80% accuracy whereas for post training quantize using nncf , we observed accuracy of 35-40%. I have not performed any fine tuning while quantization. 
Can you explain the reason for accuracy drop.?

**Comment by adrianboguszewski** [comment]
@MaximProshin, could you look at this?

**Comment by MaximProshin** [comment]
Dear @yogeshReflexionai , thanks for your interest! 
First of all, I must say that the post-training quantization doesn't always guarantee accurate results but we always try it before the fine-tuning and so sometimes we have to adjust the default post-training quantization. The simplest way to improve the accuracy of a quantized model is to exclude some layers from the quantization if you know which of them bring the most of the accuracy degradation. The example of such ignored layers can be found in yolov8 notebook, step [24]: https://github.com/openvinotoolkit/openvino_notebooks/blob/main/notebooks/230-yolov8-optimization/230-yolov8-optimization.ipynb 
Can you check if it's possible to identify such layers on your side and then exclude them? 
If you could share your code of the NNCF quantization with us, we would also check how it could be improved. 

@AlexKoff88 , I think we never tried to quantize this model before but you could probably check it and suggest some adjustments right away.   

@yogeshReflexionai , please note that this repository is for openvino notebooks, PRs and issues related to them. For NNCF related issues please use NNCF repository in the future (https://github.com/openvinotoolkit/nncf) and my team will be happy to support you.

**Comment by AlexKoff88** [comment]
No, we haven't looks at this model but most likely the problem is in the layers sensitive to quantization. They are basically at the end of the model, e.g. in the result post-processing subgraph.

**Comment by raymondlo84** [comment]
> Hi, I have observed 40-45% accuracy drop while performing post quantization on Residual Masking network. For pytorch model we have accuracy on 80%, for ir convertion we get exactly 80% accuracy whereas for post training quantize using nncf , we observed accuracy of 35-40%. I have not performed any fine tuning while quantization. [Residual Masking network ](https://github.com/phamquiluan/ResidualMaskingNetwork) Can you explain the reason for accuracy drop.?

Do you have the source code that you can share?

**Comment by adrianboguszewski** [comment]
Closing because of the lack of activity. Please reopen if needed.
